### PR TITLE
[CIS-306] Fix date separators not appearing after a new message

### LINKED ISSUE #436: Date separators don't appear in ChatViewController immediately
## What did you do?
1. Post a message in ChatViewController
2. Wait until tomorrow and then post a 2nd message
3. The date separator does not appear

## What did you expect to happen?
I expected the date separator to appear immediately after I post 2nd message

## What happened instead?
The date separator does not appear until I restart the app and re-enter the chat

## GetStream Environment
**GetStream Chat version:**
2.2.6
**GetStream Chat frameworks:** StreamChatClient, StreamChatCore, StreamChat
**iOS version:**
**Swift version:**
**Xcode version:**
**Device:**

## Additional context

gz#5574

### PR DESCRIPTION
**Pull request by b-onc**
Fixes #436

### CODE REVIEW

File: Sources/Core/Presenter/Channel/ChannelPresenter+Events.swift [hunk 1]
```diff
@@ -159,6 +166,19 @@ extension ChannelPresenter {
...
+            !lastMessage.created.isLessThan(timeInterval: 60*60*24, with: message.created),
+            message.created.isToday else {
+                return false
+        }
+        
+        items.append(.status(PresenterItem.statusTodayTitle,
+                             "at \(DateFormatter.time.string(from: message.created))", false))
+        return true
+    }
+
```
Comment (buh) [inline Sources/Core/Presenter/Channel/ChannelPresenter+Events.swift:181]: This looks like copy/paste from `ChannelPresenter+Request`. Can you use this function there too?

**Review by buh — APPROVED** [review]
LGTM. 
Please check my comment to avoid duplicated code.

File: Sources/Core/Presenter/Channel/ChannelPresenter+Events.swift (hunk 1 above)
Comment (b-onc) [reply Sources/Core/Presenter/Channel/ChannelPresenter+Events.swift:181]: Only line 177 is copy/paste, the rest cannot be reused
The part in question for easy review:
```swift
if showStatuses, !todayStatusAdded, message.created.isToday {
    todayStatusAdded = true
    items.insert(.status(PresenterItem.statusTodayTitle,
                         "at \(DateFormatter.time.string(from: message.created))",
        false), at: index)
    index += 1
}
```

Comment (buh) [reply Sources/Core/Presenter/Channel/ChannelPresenter+Events.swift:181]: Up to you ¯\_(ツ)_/¯ 

By the way, `60*60*24` = `86_400`

**Pull request merged**